dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -0,0 +1,372 @@
+﻿#include "pch.h"
```

Comment:
Missing Copyright

same comment applies throughout

Reply:
Adding in next iteration.